src/spring-cloud/azext_spring_cloud/commands.py

```diff
@@ -168,9 +202,75 @@ def load_command_table(self, _):
                             custom_command_type=service_registry_cmd_group,
                             exception_handler=handle_asc_exception,
                             is_preview=True) as g:
-        g.custom_show_command('show', 'service_registry_show', table_transformer=transform_service_registry_output)
+        g.custom_show_command('show', 'service_registry_show',
+                              table_transformer=transform_service_registry_output)
         g.custom_command('bind', 'service_registry_bind')
         g.custom_command('unbind', 'service_registry_unbind')
 
+    with self.command_group('spring-cloud application-configuration-service',
+                            custom_command_type=application_configuration_service_cmd_group,
+                            exception_handler=handle_asc_exception,
+                            is_preview=True) as g:
+        g.custom_command('clear', 'application_configuration_service_clear')
+        g.custom_show_command('show', 'application_configuration_service_show',
+                              table_transformer=transform_application_configuration_service_output)
+        g.custom_command('bind', 'application_configuration_service_bind')
+        g.custom_command('unbind', 'application_configuration_service_unbind')
+
+    with self.command_group('spring-cloud application-configuration-service git repo',
+                            custom_command_type=application_configuration_service_cmd_group,
+                            exception_handler=handle_asc_exception) as g:
+        g.custom_command('add', 'application_configuration_service_git_add')
+        g.custom_command('update', 'application_configuration_service_git_update')
+        g.custom_command('remove', 'application_configuration_service_git_remove')
```

Comment:
Do we need to consider adding `confirmation=True` for the remove command?

Reply:
Thanks for suggestion, since application configuration service git repo is not an individual sub resource and remove it is not a heavy action for customer, adding the confirmation seems not that necessary.

Reply:
OK